server/utils/nunjucksSetup.ts

```diff
@@ -119,6 +119,10 @@ export function registerNunjucks(app?: express.Express): Environment {
     return appointmentTimeType[type]
   })
 
+  njkEnv.addFilter('beforeOrAfterHardStop', (licences: Record<string, unknown>[], isHardstop: boolean) => {
+    return licences.filter(c => c.hardStop === isHardstop)
+  })
```

Comment:
I think we've talk before about the disadvantages of using boolean flags: 

`data | beforeOrAfterHardStop(true)`
vs
`data | beforeOrAfterHardStop(false)`

How are you supposed to know what each call means? 

Would probably just split it out into two separate methods